examples/src/App.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { ComponentTemplate } from '@sandstreamdev/react-component-template';
+
+import styles from './app.module.css';
```

Comment:
I am not a fan of CSS modules. For simple components, it may be an overkill and makes other contributors face the wall of unknown tech solution that yields little value over its cost.
Can we consider keeping plain CSS?